code/modules/vehicles/mecha/_mecha.dm

```diff
@@ -242,6 +242,7 @@
 	return ..()
 
 /obj/vehicle/sealed/mecha/obj_destruction()
+	explosion(loc, -1, 0, 1, 2, 3, 3)
```

Comment:
Would be funny, even more so with the fact that the pilot gets slept for 2 seconds.

Reply:
If you don't mind, i'm gonna make the explosion more lethal and give the pilot a 2 second stun and 5 second knockdown because it'd be pretty tense to have that happen to you, or see it happen.